File: solutions/Javascript/Power of Two.js
var isPowerOfTwo = function (n) {
  function helper(i, n) {
    if (Math.pow(2, i) === n) return true;
    if (Math.pow(2, i) > n) return false;
    return helper(i + 1, n);
  }
  return helper(0, n);
};

File: solutions/python/Power of Two.py
class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        def helper(i,n):
            if math.pow(2,i)==n:
                return True
            if math.pow(2,i)>n:
                return False
            return helper(i+1,n)
        return helper(0,n)